storybook/storybook-react/src/components/Link/Link.stories.tsx

```diff
@@ -106,6 +100,16 @@ export const InList: Story = {
   },
 }
 
+export const InListWithIcon: Story = {
+  args: {
+    children: 'Linktekst',
+    href: '#',
+    variant: 'inList',
+    onBackground: undefined,
+    icon: Email,
```

Comment:
No Chevron?

Reply:
Do you mean we should disallow chevrons in the `icons` prop?

Reply:
No, I mean the inList example in Figma has a chevron

Reply:
An `inList` link has a chevron by default, so the `InList` story already shows the chevron

Reply:
The chevron icon is the default, but it can be overridden.